src/main/java/de/telran/projectredirectservice/controller/RedirectRestController.java

```diff
@@ -0,0 +1,26 @@
+package de.telran.projectredirectservice.controller;
+
+import de.telran.projectredirectservice.service.RedirectService;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+public class RedirectRestController {
+
+    final RedirectService redirectService;
+
+    public RedirectRestController(RedirectService redirectService) {
+        this.redirectService = redirectService;
+    }
+
+    @PostMapping("server/urls")
+    public String createShortURLFromLong(@RequestBody String longURL) {
```

Comment:
I would consider posting richer structure, which includes a customer_number, for example.

Reply:
And may be an expiration date.